### DISCUSSION TITLE
Code Format

### DISCUSSION THREAD

**Discussion by appt2** [mozilla/rhino#1423] [Q&A]
Hello, does your library have the ability to sort codes?
```[tasklist]
### Tasks
```

**Comment by tonygermano** [comment]
It's not clear what you are asking, but this question likely belongs in Discussions and not Issues.

**Comment by appt2** [comment]
> It's not clear what you are asking, but this question likely belongs in Discussions and not Issues.

Sort JavaScript code

**Reply by tonygermano** [reply]
It supports Array.prototype.sort if that's what you are asking.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
